exercises/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,10 @@
 // Complete the function so that it takes input parameters
 function multiply() {
   // Calculate the result of the function and return it
+  function multiply(a, b) {
```

Comment:
This won't work properly, the outer function is the one that's getting called as the inner function is scoped to the inside of the curly brackets it's in

In the sample below result will equal 12 cos it's being called within the same curly brackets the inner function is defined in and the inner function is overriding the outer function
```
function multiply() {
  // Calculate the result of the function and return it
  function multiply(a, b) {
  return a*b;
  }
  let result = multiply(3, 4);
}
```
But it will be undefined in this case as it only sees the outer function which doesn't return anything
```
function multiply() {
  // Calculate the result of the function and return it
  function multiply(a, b) {
  return a*b;
  }
  
}
let result = multiply(3, 4);
```